### ISSUE TITLE
Upload the Guideline for Presentation in Good Order to the Wagtail CMS

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2995]
**What we are after:**
In preparation for the new presidential section, we need to move the Guideline for Presentation in Good Order (2007) and its appendix to the Wagtail CMS

### Related issues

This book will be a part of our drafting public funding-related presidential pages.

### Completion criteria

- [x] Download the current versions (you can find them on https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/) and update the SEO and accessibility features of the PDF files.
- [x] Upload to the CMS with a good solid URL
- [x] Update the links to them on these pages:

- https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/ 
- https://transition.fec.gov/law/policy.shtml
- https://transition.fec.gov/general/library.shtml

- [x] Add the PDFs to the [redirect spreadsheet
](https://docs.google.com/spreadsheets/d/1Jn5nC3fFku10RTCP5OAW6RiffjNBE4VwBubITQpCNK8/edit#gid=0) and the next redirect ticket.
### Future work

- Redirect in transition and www
- Put in a ticket to have front end take down the old PDFs once redirect is in place.

**Comment by dorothyyeager** [comment]
Doublechecking with the Audit Division now to ensure we have the latest version. The version currently up on our website is dated 2007.

**Comment by dorothyyeager** [comment]
The 2007 version is indeed the latest version. Working on making it ready for upload.

**Comment by dorothyyeager** [comment]
Here are the new links:

Guideline in Good Order: https://www.fec.gov/resources/cms-content/documents/guideline-for-presentation-good-order.pdf

Appendices: https://www.fec.gov/resources/cms-content/documents/guideline-for-presentation-good-order_appendices.pdf

**Comment by dorothyyeager** [comment]
The new files are in place, the old links are updated, I've sent them to Audit, and I've added the new and old URLs to our redirect ticket for 9.4 #3000 . Work on this ticket is complete so closing.
